versioned_docs/version-1.0.0/uptime/uptime.md

```diff
@@ -0,0 +1,60 @@
+---
+id: uptime-check-model
+slug: /uptime-check-model
+sidebar_position: 1
+---
+# Uptime
+
+The uptime check feature allows you to determine whether their site is currently
+online or not. If the site goes down, notifications can be sent via email or
+Slack. For more details on notifications, please refer to the
+[notification section](../notifications/notifications.md).
```

Comment:
We can use "a" instead of "their" to avoid ownership or possession on Uptime Check over Site: 

```suggestion
The uptime check feature allows you to determine whether a site is currently online or not. If the site goes down, notifications can be sent via email or Slack. For more details on notifications, please refer to the [notification section](../notifications/notifications.md).
```